src/components/DatePicker/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import moment from 'moment';
+import ExpensiTextInput from '../ExpensiTextInput';
+import CONST from '../../CONST';
+import {propTypes, defaultProps} from './datepickerPropTypes';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
+
+const webDatePickerPropTypes = {
+    ...propTypes,
+    ...windowDimensionsPropTypes,
+};
+
+class WebDatepicker extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.raiseDateChange = this.raiseDateChange.bind(this);
+
+        /* We're using uncontrolled input otherwise it wont be possible to
+        * raise change events with a date value - each change will produce a date
+        * and make us reset the text input */
+        this.defaultValue = props.value
+            ? moment(props.value).format(CONST.DATE.MOMENT_FORMAT_STRING)
+            : '';
+    }
+
+    componentDidMount() {
+        if (this.inputRef) {
+            // Adds nice native datepicker on web/desktop. Not possible to set this through props
+            this.inputRef.type = 'date';
```

Comment:
This feels like a hack/code smell to me, but I don't have a better suggestion atm. Seems like it works so we can go with it for now.

Reply:
Yep, it's hackish ⌨️ but allows us to reuse `ExpensiTextInput`
Other options are:
- don't set the type to "date" and not have the native picker for web/mWeb/desktop.
- use html elements (since we're on a web/desktop file) and return `<input type=date />`. But we'll have to refactor  or re-code all the styles to match `ExpensiTextInput` 

I did a quick search for other ways to pass this attribute to the native element and didn't find anything